### DISCUSSION TITLE
Naming for the *value* for the attribute that means "just close with ESC"

### DISCUSSION THREAD

**Discussion by mfreed7** [openui/open-ui#961] [Polls]
See https://github.com/openui/open-ui/discussions/950 for the first incarnation of this poll. After some [additional discussion in WHATWG](https://github.com/whatwg/html/issues/9373#issuecomment-1830376481), we've coalesced around an attribute with multiple values, such as:

- `<dialog closetrigger=none>` - no automatic closing at all
- `<dialog closetrigger=attribute_value_here>` - closes via ESC (or any such close signal)
- `<dialog closetrigger=any>` - closes via ESC *or* clicking outside the bounds of the dialog

This poll is about "`attribute_value_here`" in bullet <span>#</span>2 in the above list. What should that value be called?

(Note that the naming for "`closetrigger`" is a [separate poll](https://github.com/openui/open-ui/discussions/960). This one is just about the value for ESC / close signal.)

**Comment by GrimLink** [comment]
The term `closerequest` appears to be redundant, as the attribute already employs the word close in its name, as seen in the example `<dialog closetrigger=closerequest>`. For improved clarity, it would be more appropriate to simply use `request` as the value for the ESC / close signal."

**Comment by zcorpan** [comment]
Something else: `esc`

While it could seem device-specific, I think it's probably understandable enough and not surprising that equivalent signals on non-keyboard devices also work. It's also concise and easy to type.

**Reply by mfreed7** [reply]
This isn't a bad suggestion. I agree it's "close enough" to a close request and is likely almost more understandable than "close request".

I just tried to add this as another poll item, but apparently that would clear existing votes. So how about this: if you like this option (`esc`) do not vote above, and instead give this comment a 👍 .
